### PR TITLE
optimize: console log optimize for "kubectl logs -f"

### PR DESCRIPTION
**Pull request by PeppaO**
- [ ] I have registered the PR [changes](../changes).

### Ⅰ. Describe what this PR did
仅输出启动日志到控制台便于启动报错排查
fixes #6152

### CODE REVIEW

**Review by funky-eyes — COMMENTED** [review]
实现的有些粗糙，我希望有更优的解决方案
The implementation is a bit rough, I hope there is a better solution

File: changes/en-us/2.x.md
Comment (funky-eyes) [inline changes/en-us/2.x.md:1]: 作者信息还未登记
The author's information has not been registered yet

Comment (PeppaO) [reply changes/en-us/2.x.md:1]: done

**Review by funky-eyes — DISMISSED** [review]
LGTM

**Review by funky-eyes — APPROVED** [review]
LGTM

**Review by xingfudeshi — APPROVED** [review]
LGTM

**Review by slievrly — COMMENTED** [review]
LGTM

**Pull request merged**